src/model_theory/terms_and_formulas.lean

```diff
@@ -362,6 +379,137 @@ begin
       { exact (dif_neg (λ h', h (nat.lt_of_add_lt_add_left h'))).trans (dif_neg h).symm } } }
 end
 
+/-- An atomic formula is either equality or a relation symbol applied to terms. -/
+inductive is_atomic : L.bounded_formula α n → Prop
```

Comment:
It depends on convention whether `⊥` is atomic or not. Please mention that it is not atomic here in the docstring.